api/envoy/config/trace/v2/trace.proto

```diff
@@ -184,6 +184,13 @@ message OpenCensusConfig {
   // https://github.com/grpc/grpc/blob/master/doc/naming.md
   string stackdriver_address = 10;
 
+  // (optional) The call credentials used in gRPC call to production Stackdriver.
+  // Currently only :ref:`sts_service <envoy_v3_api_field_config.core.v3.GrpcService.GoogleGrpc.CallCredentials.sts_service>`
+  // is supported by opencensus tracer. If not provided, or the provided option is not
```

Comment:
From the code it looks like default credentials are used in addition to STS. Is that what gRPC normally does?

Reply:
Yeah oc library uses default credential for both channel and call credentials. https://github.com/census-instrumentation/opencensus-cpp/blob/c900c4d723ef596f357b0e695e68e2fa725eec90/opencensus/exporters/stats/stackdriver/internal/stackdriver_exporter.cc#L91

This change make it use sts for call credentials but still uses google default for channel credentials.